pkg/skaffold/runner/new.go

```diff
@@ -177,16 +178,17 @@ func getBuilder(runCtx *runcontext.RunContext) (build.Builder, bool, error) {
 		if err != nil {
 			return nil, false, err
 		}
+		builder.WithArtifactStore(store)
```

Comment:
This is an example of `With` not following the builder pattern.

Reply:
will refactor from your above comment.

Reply:
done.